<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inviter</class>
 <widget class="QDialog" name="Inviter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invite sender</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Invite">
    <attribute name="title">
     <string>Invite</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>321</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Send invitation to:</string>
     </property>
     <widget class="QLabel" name="mail">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Emails:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="subject">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Subject:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSubject">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="body">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Body:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="leBody">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="leEmails">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="leCount">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="leStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> No data.</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btSend">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>PDF Content</string>
    </attribute>
    <widget class="QPushButton" name="btSavePDF">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>321</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>PDF content:</string>
     </property>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Event title:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leDate">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="date">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Event date:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leLocation">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="location">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Event location:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leEvent">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="accountSettings">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QPushButton" name="btSave">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Log file path :</string>
     </property>
     <widget class="QLineEdit" name="lePath">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btPath">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>321</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Mail settings:</string>
     </property>
     <widget class="QLabel" name="port">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>port:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePasswd">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="passwd">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePort">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="login">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Login:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="leLogin">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leServer">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="server">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SMTP server:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leServer</tabstop>
  <tabstop>lePort</tabstop>
  <tabstop>leLogin</tabstop>
  <tabstop>lePasswd</tabstop>
  <tabstop>btSave</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>leEmails</tabstop>
  <tabstop>leSubject</tabstop>
  <tabstop>leBody</tabstop>
  <tabstop>btSend</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
